pkg/run/run.go

```diff
@@ -34,14 +36,19 @@ import (
 )
 
 type RunOptions struct {
-	plugins       *[]string
-	dedicated     bool
-	sonobuoyImage string
-	timeout       int
-	watch         bool
-	devCount      string
-	mode          string
-	upgradeImage  string
+	plugins         *[]string
+	dedicated       bool
+	sonobuoyImage   string
+	imageRepository string
+	// ToolsImage
+	// defines the image containing plugins associated with the provider-certification-tool.
+	// this variable is referenced by plugin manifest templates to dynamically reference the tools image.
+	ToolsImage   string
```

Comment:
What about using something specific with the plugin, maybe `PluginImage`, `PluginCertImage`, or something related?